packages/protocol-kit/tests/e2e/contractInfo.test.ts

```diff
@@ -0,0 +1,98 @@
+import { setupTests } from '@safe-global/testing-kit'
+import Safe from '@safe-global/protocol-kit/index'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import { getEip1193Provider } from './utils/setupProvider'
+
+chai.use(chaiAsPromised)
+
+describe('Contract Info', () => {
+  const provider = getEip1193Provider()
+  let protocolKit: Safe
+
+  beforeEach(async () => {
+    const { safe, contractNetworks } = await setupTests()
+    const safeAddress = safe.address
+    protocolKit = await Safe.init({
```

Comment:
Why do you need a new instance before each test?

Reply:
don't need